pkgs/development/python-modules/opytimark/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonPackage
+, coverage
+, fetchFromGitHub
+, numpy
+, pre-commit
+, pylint
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "opytimizer";
+  version = "1.0.8";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "gugarosa";
+    repo = "opytimark";
+    rev = "v${version}";
+    hash = "sha256-T3OFm10gvGrUXAAHOnO0Zv1nWrXPBXSmEWnbJxrWYU0=";
+  };
+
+  nativeBuildInputs = [ coverage pre-commit pylint ];
```

Comment:
Those modules are relevant for developing and are not requirements. Please remove them by patching upstream's `setup.py`.